### ISSUE TITLE
Implement Sonar antibot support

### ISSUE THREAD

**Issue by overkidding** [kyngs/LibreLogin#205]
`
[08:43:38 ERROR] [LibreLogin]: The uuid field was not found in the PendingConnection class, please report this to the developer. And attach the class summary below.
[08:43:38 ERROR] [LibreLogin]: -- BEGIN CLASS SUMMARY --
[08:43:38 ERROR] [LibreLogin]: Class: xyz.jonesdev.sonar.bungee.fallback.FallbackInitialHandler
[08:43:38 ERROR] [LibreLogin]: net.md_5.bungee.netty.ChannelWrapper: channelWrapper
[08:43:38 ERROR] [LibreLogin]: net.md_5.bungee.BungeeCord: bungee
[08:43:38 ERROR] [LibreLogin]: xyz.jonesdev.sonar.bungee.fallback.FallbackUserWrapper: user
[08:43:38 ERROR] [LibreLogin]: xyz.jonesdev.sonar.api.fallback.protocol.ProtocolVersion: protocolVersion
[08:43:38 ERROR] [LibreLogin]: boolean: receivedLoginPacket
[08:43:38 ERROR] [LibreLogin]: boolean: receivedStatusPacket
[08:43:38 ERROR] [LibreLogin]: xyz.jonesdev.sonar.api.fallback.Fallback: FALLBACK
[08:43:38 ERROR] [LibreLogin]: java.util.Map: CACHED_KICK_PACKETS
[08:43:38 ERROR] [LibreLogin]: -- END CLASS SUMMARY --
`

**Comment by kyngs** [comment]
Sanely formatted log:
```
[08:43:38 ERROR] [LibreLogin]: The uuid field was not found in the PendingConnection class, please report this to the developer. And attach the class summary below. [08:43:38 ERROR] [LibreLogin]: -- BEGIN CLASS SUMMARY -- 
[08:43:38 ERROR] [LibreLogin]: Class: xyz.jonesdev.sonar.bungee.fallback.FallbackInitialHandler 
[08:43:38 ERROR] [LibreLogin]: net.md_5.bungee.netty.ChannelWrapper: channelWrapper 
[08:43:38 ERROR] [LibreLogin]: net.md_5.bungee.BungeeCord: bungee 
[08:43:38 ERROR] [LibreLogin]: xyz.jonesdev.sonar.bungee.fallback.FallbackUserWrapper: user 
[08:43:38 ERROR] [LibreLogin]: xyz.jonesdev.sonar.api.fallback.protocol.ProtocolVersion: protocolVersion 
[08:43:38 ERROR] [LibreLogin]: boolean: receivedLoginPacket 
[08:43:38 ERROR] [LibreLogin]: boolean: receivedStatusPacket 
[08:43:38 ERROR] [LibreLogin]: xyz.jonesdev.sonar.api.fallback.Fallback: FALLBACK 
[08:43:38 ERROR] [LibreLogin]: java.util.Map: CACHED_KICK_PACKETS 
[08:43:38 ERROR] [LibreLogin]: -- END CLASS SUMMARY --
```

**Comment by kyngs** [comment]
Honestly, this would be better if it were handled on Sonar's side, but if anyone wants to implement this, I'll accept a PR.

**Comment by overkidding** [comment]
Sonar developer replied to my issue and said that will fix ASAP

**Comment by overkidding** [comment]
Patched by [#164](https://github.com/jonesdevelopment/sonar/pull/164)
